pkg/healthcheck/health_controller.go

```diff
@@ -138,7 +144,7 @@ func (hc *HealthController) CheckHealth() bool {
 }
 
 //RunServer starts the HealthController's server
-func (hc *HealthController) RunServer(stopCh <-chan struct{}, wg *sync.WaitGroup) error {
+func (hc *HealthController) RunServer(stopCh <-chan struct{}, wg *sync.WaitGroup) {
```

Comment:
Removed error return value here since it was not used and it's a go routine, so probably an error channel makes more sense.